aries-site/src/pages/components/card.mdx

```diff
@@ -163,15 +347,23 @@ These are example layouts for various use cases.
   <MarketplaceNavigationalCards />
```

Comment:
I know this section wasn't touched in this PR, but I did notice that the spacing of these examples makes it look like the show example code button is associated with the example below rather than the one above. In the picture 'Show Example Code' looks associated with 'Activities and Tasks' rather than 'Applications'
<img width="1258" alt="Screen Shot 2022-09-27 at 1 35 59 PM" src="https://user-images.githubusercontent.com/54560994/192619818-7f342fef-1867-4fe3-abc8-20e97c5410df.png">

Reply:
Great note, I agree. I might add a heading before each example to remedy this and allow people to deep link directly to a given example.

Reply:
Done.